docs/modules/operation/pages/deep-dive/admin/webui/dashboard.adoc

```diff
@@ -1,137 +1,154 @@
-5. Last Time: Time of the most recent alarm.
-6. Log Msg: The log message from the event definition.
+.An alarms component displaying three alarms--two of Minor severity, and one of Warning severity
+image::dashboard/03_dashboard-alarms.png[]
 
-The Alarms component shows the most recent alarms and lets the user scroll through the last 100 alarms.
+1. *Node:* the label of the node associated with the alarm.
+2. *Severity:* severity of the alarm.
+3. *UEI:* the alarm's UEI.
+4. *Count:* the number of alarms deduplicated by the alarm's reduction key.
+5. *Last Time:* the time at which the most recent alarm was generated.
+6. *Log Msg:* the log message from the alarm's event definition.
 
 [[ga-dashboard-notifications]]
 === Notifications
 
-To inform people on a duty schedule notifications are used and force action to fix or reconfigure systems immediately.
-In {page-component-title} it is possible to acknowledge notifications to see who is working on a specific issue.
-The dashboard should show outstanding notifications in the NOC to provide an overview and give the possibility for intervention.
+The notifications component shows outstanding and unacknowledged notifications.
+It sorts notifications from most to least recent, and lets you scroll through the last 100 notifications.
 
-.Information displayed in the Notifications component
-image::dashboard/04_dashboard-notifications.png[]
+Notifications are used to inform people on a duty schedule of issues in the monitored network, and to force action to fix or reconfigure systems immediately.
+In {page-component-title}, it is possible to acknowledge notifications to designate someone as working on a specific issue.
+
+A dashboard should show outstanding notifications in the NOC.
+This provides an overview of the monitored network and gives an opportunity for intervention.
+
+The following information is shown in the component:
 
-1. Node: Label of the node associated with the notification.
-2. Service: Name of the service associated with the notification.
-3. Message: Message associated with the notification.
-4. Sent Time: Time when the notification was sent.
-5. Responder: User name who acknowledged the notification.
-6. Response Time: Time when the user acknowledged the notification.
+.A notifications component displaying three notifications
+image::dashboard/04_dashboard-notifications.png[]
 
-The Notifications component shows the most recent unacknowledged notifications and lets the user scroll through the last 100 notifications.
+1. *Node:* the label of the node associated with the notification.
+2. *Service:* the name of the service associated with the notification.
+3. *Message:* the log message associated with the notification.
+4. *Sent Time:* time when the notification was sent.
+5. *Responder:* name of the user who acknowledged the notification.
+6. *Respond Time:* the time when the Responder acknowledged the notification.
```

Comment:
```suggestion
* *Respond Time:* the time when the responder acknowledged the notification.
```
I wish the UI said "response time".

Reply:
Same. Maybe we can review the UI language at some point?